core/src/main/java/com/graphhopper/routing/util/CarFlagEncoder.java

```diff
@@ -158,6 +158,9 @@ public int defineWayBits(int index, int shift) {
 
     protected double getSpeed(ReaderWay way) {
         String highwayValue = way.getTag("highway");
+        if (way.hasTag("motorroad", "yes")) {
```

Comment:
Furthermore would you mind to send me an email regarding [our CLA](https://github.com/graphhopper/graphhopper/blob/master/.github/CONTRIBUTING.md#license-agreement)?